### PR TITLE
Championship slot

### PR DESCRIPTION
**Pull request by axelclark**
Add championship slots to championship show page.  Also add fantasy team to owner column.

### CODE REVIEW

File: web/admin/championship_slot.ex [hunk 1]
```diff
@@ -0,0 +1,7 @@
+defmodule Ex338.ExAdmin.ChampionshipSlot do
```
Comment (credoci) [inline web/admin/championship_slot.ex]: Modules should have a @moduledoc tag.

File: web/models/championship_slot.ex [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    belongs_to :roster_position, Ex338.RosterPosition
+    belongs_to :championship, Ex338.Championship
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
```
Comment (credoci) [inline web/models/championship_slot.ex]: Parameter `struct` has same name as the `Kernel.struct` function.

File: web/models/championship_slot.ex [hunk 3]
```diff
@@ -0,0 +1,20 @@
+defmodule Ex338.ChampionshipSlot do
```
Comment (credoci) [inline web/models/championship_slot.ex]: Modules should have a @moduledoc tag.

File: web/controllers/championship_controller.ex [hunk 4]
```diff
@@ -10,9 +10,13 @@ defmodule Ex338.ChampionshipController do
...
 
   def show(conn, %{"fantasy_league_id" => league_id, "id" => id}) do
+    league = FantasyLeague.get_league(league_id)
+    championship =
+      Championship.get_championship_by_league(Championship, id, league_id)
+
     render(conn, "show.html",
-      championship:   Championship.get_championship(Championship, id),
-      fantasy_league: FantasyLeague.get_league(league_id)
+      championship: championship,
```
Comment (axelclark) [inline web/controllers/championship_controller.ex]: Not necessary, can get league_id from parameters

File: web/models/championship.ex [hunk 5]
```diff
@@ -41,6 +41,12 @@ defmodule Ex338.Championship do
     |> Repo.all
   end
 
+  def get_championship_by_league(query, id, league_id) do
```
Comment (axelclark) [inline web/models/championship.ex]: Need test

File: web/models/championship_result.ex [hunk 6]
```diff
@@ -30,12 +32,25 @@ defmodule Ex338.ChampionshipResult do
...
+      preload: [fantasy_player: ^player]
+  end
+
   def get_assocs_and_order_results(query) do
     query
     |> preload_assocs
     |> order_by_rank
   end
 
+  def get_ordered_assocs_by_league(query, league_id) do
```
Comment (axelclark) [inline web/models/championship_result.ex]: Need test

File: web/views/championship_view.ex [hunk 7]
```diff
@@ -1,3 +1,10 @@
 defmodule Ex338.ChampionshipView do
   use Ex338.Web, :view
+
+  def get_team_name(%{fantasy_player: %{roster_positions: [position]}}) do
```
Comment (axelclark) [inline web/views/championship_view.ex]: Need test

**Pull request merged**